docs/release.rst

```diff
@@ -110,3 +110,19 @@ When a new Python major version is released
 - Update the list of versions in ``tools/build/build_macos_arm64.sh`` to include
   the new version. Look for both the ``python_versions`` variable and the
   ``CIBW_BUILD`` environment variable.
+
+
+When dropping end-of-life Python versions
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+- Update project metadata, ``requires-python`` and (maybe) package dependencies
+  in ``pyproject.toml`` files of the corresponding ``psycopg`` directories
+
+- Update necessary GitHub Actions workflow files in the ``.github/workflows/`` directory,
+  e.g., ``tests.yml`` and ``.3rd-party-tests.yml`` and so forth.
+
+- Bump needed versions in the ``tests/constraints.txt`` file.
+
+Examples:
+
+- `PR #977 <https://github.com/psycopg/psycopg/pull/977>`_
```

Comment:
Looks good? Waiting for your suggestions, will be added ASAP.

Reply:
I would add (unless you want to do it in a different MR):

```
- Grep the source code for occurrences of the version to be dropped and the
  version following to find code branches depending on the version to remove.
  A few examples (dropping Python 3.8)::

     git grep -E -e '\b3\.8\b' -e '\b(cp)?38\b' -e '\b3, 8\b'
     git grep -E -e '\b3\.9\b' -e '\b(cp)?39\b' -e '\b3, 9\b'
```

Reply:
Changed, is it still good?